configure.ac

```diff
@@ -327,6 +327,7 @@ AC_CONFIG_FILES([Makefile
 		targets/psa_switch/Makefile
 		targets/psa_switch/tests/Makefile
 		targets/pna_nic/Makefile
+        targets/pna_nic/tests/Makefile
```

Comment:
Nit: format to align indentation?

Reply:
Aligned it.
tabs are used for indentation in configuration files. Everywhere else, spaces are used.